### PR TITLE
Support for Login with Email Address and Link

### PR DESCRIPTION
**Pull request by oroghene**
I added a `views` directory to contain pages for `Home`, `Login`, `LoginUrl`, and `Success`. 
`Login` has the form where the user inputs their email address. 
`LoginUrl` is where users are directed to after signing in with the link sent to their email accounts.
`Success` shows confirmation that the link has been sent to the user's email address.
I added `App.js` because I did not know how else to structure the files.
I created three functions in `functions.js`: sendLoginLink which sends the login link to the user's email, catchLoginLink which checks if the user signed in using login link and redirects to `Home` if so. The last function is `logout`. 
I changed `Auth` to return the children components unconditionally.

### CODE REVIEW

**Review by ryanmarshallmc — APPROVED** [review]
Hey Roghe - great work on this! I just pushed a commit to add to your pull request that moves your code into the existing file structure. Let's find some time tomorrow to talk through the specific changes I made, and why.

TLDR: you had all the right code! It's just a matter of structuring what goes where in a React app. These changes won't seem significant now, but will definitely make the app easier to maintain as the codebase grows.

Again, great work. An AWESOME first project, I am blown away! 👍

**Comment by ryanmarshallmc** [comment]
I just added Slack integration, so updates on this should now appear in our slack!
